### DISCUSSION TITLE
Table Layout — Ability to Freeze Left Column

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#7232] [Feature Requests]
This feature would allow freezing 1+ columns on the left of the Table layout, so that it doesn't follow the x-scroll... but instead, stays visible. Not sure if this should apply on mobile (or smaller viewports), but a nice feature.

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
